README.md

```diff
@@ -1 +1,27 @@
-# linter-configure
\ No newline at end of file
+# Hello Microverse
```

Comment:
# Congratulaton on your first pull request 👏🎉

- [ ] Welcome to microverse 🥳. Good job so far 👍. Please edit the description and add a text what have you done on this pull request. This will be very helpful also in future projects 👋.

This is the place where you should edit
![image](https://user-images.githubusercontent.com/69011963/152960463-7ea89243-0f5f-4ac2-9f84-4a274a702a68.png)